### PR TITLE
enabled storage of user-labeled data

### PR DESCRIPTION
**Pull request by jayliu223**
An alert message tells the user that their feedback is being stored. 

In the backend, an if/else statement is used to squash data to 0 or1 before storing the label with the comment in DB.

image

### CODE REVIEW

**Review by MahreenAsghar — APPROVED** [review]
Good Job Jay on implementing this feature! Now we will have more accurate Sentiment analysis!!

**Comment by jayliu223** [comment]
This won't improve prediction accuracy on the fly, but it does give us a growing list of labeled training data that can be used to fine-tune our model.
